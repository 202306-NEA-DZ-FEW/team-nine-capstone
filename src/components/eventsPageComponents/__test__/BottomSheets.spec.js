import renderer from "react-test-renderer";

import BottomSheet from "../BottomSheets";

describe("BottomSheet Snapshot", () => {
    it("should matches DOM Snapshot", () => {
        const tree = renderer.create(<BottomSheet />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
