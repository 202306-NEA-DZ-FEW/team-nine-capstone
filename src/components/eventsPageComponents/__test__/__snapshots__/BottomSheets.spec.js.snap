// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BottomSheet Snapshot should matches DOM Snapshot 1`] = `
<div
  className="fixed h-80 rounded-t-lg bottom-0 left-0 right-0 bg-white shadow-lg transform transition-transform duration-300 translate-y-full"
>
  <div
    className="flex justify-center h-5 cursor-pointer"
  >
    <div
      className=" mt-0 w-[30%] h-2 text-center bg-gray-400 hover:bg-black rounded-xl"
    />
  </div>
  <div
    className="p-1 h-full"
  >
    <div
      className="h-72 scroll-m-4 overflow-y-auto mt-2"
    />
  </div>
</div>
`;
